Resources/Public/JavaScript/ProductManager.LazyLoading.js

```diff
@@ -113,14 +117,43 @@
 			// Jquery objects
 			$wrapper = $(settings.wrapper);
 			$loaderOverlay = $(settings.loaderOverlay);
-			$lastItem = $wrapper.find(settings.item).last();
+			//$lastItem = $wrapper.find(settings.item).last(); It's empty on first load
```

Comment:
@anjeylink do you need this commented line here?

Reply:
no